Intermittent test failures for py27

*cfn-lint version: (`cfn-lint --version`)*
0.8.2

*Description of issue.*
TravisCI/tox tests fail intermittently:

```
FAIL: test_file_negative_alias (rules.resources.updatepolicy.test_configuration.TestConfiguration)
Test failure
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/cmmeyer/Documents/DevAdvocate/cfn-python-lint/test/rules/resources/updatepolicy/test_configuration.py", line 37, in test_file_negative_alias
    self.helper_file_negative('fixtures/templates/bad/resources/updatepolicy/config.yaml', 10)
  File "/Users/cmmeyer/Documents/DevAdvocate/cfn-python-lint/test/rules/__init__.py", line 52, in helper_file_negative
    self.assertEqual(err_count, len(errs))
AssertionError: 10 != 1
```

Ok on re-run.

Please provide as much information as possible:
* Template linting issues:
  * Please provide a CloudFormation sample that generated the issue.
  * If present, please add links to the (official) documentation for clarification.
* Feature request:
  * Please provide argumentation about the missing feature. Context is key!

--- a/src/cfnlint/rules/resources/updatepolicy/Configuration.py
+++ b/src/cfnlint/rules/resources/updatepolicy/Configuration.py
@@ -210,6 +210,6 @@ class Configuration(CloudFormationLintRule):
             else:
                 message = 'UpdatePolicy should be an object'
                 matches.append(
-                    RuleMatch(['Resources', r_name, 'UpdatePolicy'], message.format(path[-1])))
+                    RuleMatch(['Resources', r_name, 'UpdatePolicy'], message.format()))
 
         return matches

